src/OpenSpaceToolkit/Astrodynamics/Trajectory/Orbit.cpp

```diff
@@ -415,6 +318,55 @@ Pass Orbit::getPassWithRevolutionNumber(const Integer& aRevolutionNumber, const
     return currentPass;
 }
 
+Array<Pass> Orbit::getPassesWithinInterval(const Interval& anInterval) const
+{
+    if (!this->isDefined())
+    {
+        throw ostk::core::error::runtime::Undefined("Orbit");
+    }
+
+    if (!anInterval.isDefined())
+    {
+        throw ostk::core::error::runtime::Undefined("Interval");
+    }
+
+    Instant currentInstant = anInterval.getStart();
+
+    Integer revolutionNumber = Integer::Undefined();
+
+    // Check if the Pass is already within Pass map
+    for (const auto& passPair : this->passMap_)
```

Comment:
```suggestion
    for (const auto& passPair : passMap_)
```
Shouldn't need this-> here cause of the unique naming. Can apply to the rest of the file or just ignore this comment so that its consistent

Reply:
I'd rather reformat the repo to either use `this` or not use `this` when referring to private variables consistently outside of this MR.